src/libraries/filcrypto/filproofs/algorithms.go

```diff
@@ -35,6 +35,8 @@ func WinSDRParams(cfg SDRCfg) *WinStackedDRG_I {
 	const NODE_SIZE = 32
```

Comment:
nit would be awesome to have a proofs const file

Reply:
Can move things to a separate file as part of some ubercalc reconciliation pass in the future. FWIW, the reason constants are being defined locally here is that there could be more than one top-level type using them (in fact it's changed several time times since the code was first written), and prematurely using variable names for namespacing seemed time-consuming and error-prone. We can lock things down when they are fully stabilized.